packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -645,17 +645,23 @@ export default class LocusInfo extends EventsScope {
   updateControls(controls) {
     if (controls && !isEqual(this.controls, controls)) {
       this.parsedLocus.controls = ControlsUtils.parse(controls);
-      const {updates, current} = ControlsUtils.getControls(this.controls, controls);
+      const {updates: {hasRecordingChanged, hasRecordingPausedChanged}, current} = ControlsUtils.getControls(this.controls, controls);
 
-      if (updates.hasRecordingChanged || updates.hasRecordingPausedChanged) {
+      if (hasRecordingChanged || hasRecordingPausedChanged) {
         let state = null;
 
-        if (updates.hasRecordingChanged) {
+        // Ordered IF Statements!
+        // `idle` > `paused` > `recording` > `resumed`
+        if (hasRecordingChanged) {
           state = current.record.recording ? RECORDING_STATE.RECORDING : RECORDING_STATE.IDLE;
         }
-        else if (updates.hasRecordingPausedChanged) {
```

Comment:
@akoushke  do you get recording as true when recording and paused is a additional values on top of recording !?

Reply:
yes